docs/conceptual/array.contains['t]-function-[fsharp].md

```diff
@@ -0,0 +1,75 @@
+---
+title: Array.contains<'T> Function (F#)
+description: Array.contains<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Array.contains<'T> Function (F#)
+
+Evaluates to `true` if the given element is in the given array.
```

Comment:
I tried to follow the wording of the documentation of the Set.contains function, which is: `Evaluates to **true** if the given element is in the given set.` However, perhaps the wording from the Map.containskey function be more appropriate: `Tests if an element is in the domain of the map.
`